app/src/pages/projects/ProjectsPage.tsx

```diff
@@ -125,7 +161,16 @@ export function ProjectsPageContent({ timeRange }: { timeRange: TimeRange }) {
   );
 
   return (
-    <Flex direction="column" flex="1 1 auto">
+    <div
+      css={css`
+        flex: 1 1 auto;
+        flex-direction: column;
+        overflow: auto;
+        padding-bottom: var(--ac-global-dimension-size-750);
```

Comment:
```suggestion
        overflow-y: auto;
        overflow-x: hidden;
```

Reply:
addressed in c88eab7d33182a95cc847bbd06b31e3a4cd9f1bc